<?xml version="1.0" ?>
<robot name="human_real_size">

<<!--############################################################################################-->
<link name="base_link"/>
<link name="base_link_connector"/>

<<!--############################################################################################-->
<link name="link_00_b_body">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/body_2.dae" scale="0.4 0.35 0.31"/>				
		</geometry>
		<origin xyz="0 0 0.1"/>	
	</visual>		
</link>
<link name="link_01_b_head">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/head.dae" scale="0.035 0.03 0.035"/>			
		</geometry>
		<origin xyz="0.03 0 0.06" rpy="0 0 1.57"/>	
	</visual>
</link>

<<!--############################################################################################-->
<link name="link_02_l_upper_arm">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/upper_arm.dae" scale="0.3 0.3 0.3"/>			
		</geometry>
		<origin xyz="0 -0.01 -0.32" rpy="0 0 0"/>		
	</visual>	
</link>
<link name="link_03_l_lower_arm">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/lower_arm.dae" scale="0.3 0.3 0.3"/>			
		</geometry>
		<origin xyz="0 -0.01 -0.3" rpy="0 0 0"/>	
	</visual>
</link>
<link name="link_04_l_hand">
	<visual>
		<geometry>	
			<mesh filename="package://human_cognition/model/meshes/hand_left.dae" scale="0.11 0.11 0.11"/>		
		</geometry>
		<origin xyz="0 0 -0.08" rpy="1.57 3.14 0"/>			
	</visual>	
</link>
<link name="link_05_l_upper_leg">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/upper_leg_left.dae" scale="0.4 0.4 0.5"/>		
		</geometry>
		<origin xyz="0 0 -0.58" rpy="0 0 0"/>	
	</visual>	
</link>
<link name="link_06_l_lower_leg">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/lower_leg.dae" scale="0.45 0.45 0.38"/>			
		</geometry>
		<origin xyz="0 0 -0.375" rpy="0 0 0"/>			
	</visual>
</link>
<link name="link_07_l_foot">
	<visual>
		<geometry>			
			<mesh filename="package://human_cognition/model/meshes/foot_left.dae" scale="0.2 0.2 0.2"/>
		</geometry>     
		<origin xyz="0.17 0.05 0.01" rpy="0 0 1.57"/>			
	</visual>
</link>

<<!--############################################################################################-->
<link name="link_08_r_upper_arm">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/upper_arm.dae" scale="0.3 0.3 0.3"/>			
		</geometry>
		<origin xyz="0 0.01 -0.32" rpy="0 0 0"/>		
	</visual>
</link>
<link name="link_09_r_lower_arm">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/lower_arm.dae" scale="0.3 0.3 0.3"/>			
		</geometry>
		<origin xyz="0 0.01 -0.3" rpy="0 0 0"/>		
	</visual>
</link>
<link name="link_10_r_hand">
	<visual>
		<geometry>			
				<mesh filename="package://human_cognition/model/meshes/hand_right.dae" scale="0.11 0.11 0.11"/>		
		</geometry>
		<origin xyz="0 0 -0.08" rpy="-1.57 0 0"/>		
	</visual>
</link>
<link name="link_11_r_upper_leg">
<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/upper_leg_right.dae" scale="0.4 0.4 0.5"/>		
		</geometry>
		<origin xyz="0 0 -0.58" rpy="0 0 0"/>	
	</visual>	
</link>
<link name="link_12_r_lower_leg">
	<visual>
		<geometry>
			<mesh filename="package://human_cognition/model/meshes/lower_leg.dae" scale="0.45 0.45 0.38"/>			
		</geometry>
		<origin xyz="0 0 -0.375" rpy="0 0 0"/>			
	</visual>
</link>
<link name="link_13_r_foot">
	<visual>
		<geometry>			
			<mesh filename="package://human_cognition/model/meshes/foot_right.dae" scale="0.2 0.2 0.2"/>
		</geometry>     
		<origin xyz="0.17 -0.05 0.01" rpy="0 0 1.57"/>			
	</visual>
</link>

<<!--############################################################################################-->
<joint name="joint_base_connector" type="fixed">
	<origin xyz="0 0 1"/>
	<parent link="base_link"/>
	<child link="base_link_connector"/>
</joint>

<<!--############################################################################################-->
<joint name="joint_00_b_core" type="continuous">
	<origin xyz="0 0 0"/>	
	<parent link="base_link_connector"/>
	<child link="link_00_b_body"/>	
</joint>

<joint name="joint_01_b_neck" type="continuous">
	<origin xyz="0 0 0.53"/>		
	<parent link="link_00_b_body"/>
	<child link="link_01_b_head"/>		
</joint>

<<!--############################################################################################-->
<joint name="joint_02_l_shoulder" type="continuous">
	<origin xyz="0 0.3 .52"/>
	<parent link="link_00_b_body"/>
	<child link="link_02_l_upper_arm"/>	
</joint>
<joint name="joint_03_l_elbow" type="continuous">
	<origin xyz="0 0 -0.25"/>
	<parent link="link_02_l_upper_arm"/>
	<child link="link_03_l_lower_arm"/>
</joint>
<joint name="joint_04_l_wrist" type="continuous">
	<origin xyz="0 0 -0.27"/>
	<parent link="link_03_l_lower_arm"/>
	<child link="link_04_l_hand"/>	
</joint>
<joint name="joint_05_l_hip" type="continuous">
	<origin xyz="0 0.1 0"/>
	<parent link="base_link_connector"/>
	<child link="link_05_l_upper_leg"/>	
</joint>
<joint name="joint_06_l_knee" type="continuous">
	<origin xyz="0 0 -0.55"/>
	<parent link="link_05_l_upper_leg"/>
	<child link="link_06_l_lower_leg"/>	
</joint>
<joint name="joint_07_l_ankle" type="continuous">
	<origin xyz="0 0 -0.45"/>
	<parent link="link_06_l_lower_leg"/>
	<child link="link_07_l_foot"/>
</joint>

<<!--############################################################################################-->
<joint name="joint_08_r_shoulder" type="continuous">
	<origin xyz="0 -0.3 0.52"/>
	<parent link="link_00_b_body"/>
	<child link="link_08_r_upper_arm"/>	
</joint>
<joint name="joint_09_r_elbow" type="continuous">
	<origin xyz="0 0 -0.25"/>
	<parent link="link_08_r_upper_arm"/>
	<child link="link_09_r_lower_arm"/>	
</joint>
<joint name="joint_10_r_wrist" type="continuous">
	<origin xyz="0 0 -0.27"/>
	<parent link="link_09_r_lower_arm"/>
	<child link="link_10_r_hand"/>	
</joint>
<joint name="joint_11_r_hip" type="continuous">
	<origin xyz="0 -0.1 0"/>
	<parent link="base_link_connector"/>
	<child link="link_11_r_upper_leg"/>	
</joint>
<joint name="joint_12_r_knee" type="continuous">
	<origin xyz="0 0 -0.55"/>
	<parent link="link_11_r_upper_leg"/>
	<child link="link_12_r_lower_leg"/>	
</joint>
<joint name="joint_13_r_ankle" type="continuous">
	<origin xyz="0 0 -0.45"/>
	<parent link="link_12_r_lower_leg"/>
	<child link="link_13_r_foot"/>	
</joint>

</robot>